MetaMask/action-npm-publish
# housekeeping

I was taking a closer look at this last night and wanted to button up a few things:
- update to use latest stable
- test the full skip functionality in ci after standardising this to use latest stable
- standardise on `yarn` for test workflow
- <strike>previously the skip functionality only worked if the module version you were trying to publish was equal to the currently published version (which was honestly probably fine), but here I've made a change so it will also skip if the version you're trying to publish is even older than that.</strike>
- <strike>I've also updated the notice for that scenario so it surfaces a bit more details</strike>
- add `private: true`
- add `.nvmrc`
you can see an example of this in the following workflow run: https://github.com/rickycodes/action-npm-publish/runs/7412228382?check_suite_focus=true
